docs/_book.yaml

```diff
@@ -68,6 +68,11 @@ upper_tabs:
       - title: "Operators and observables"
         path: /cirq/operators_and_observables
 
+      ####  Support  ####
+      - heading: "Support"
+      - title: "Issues / requests / questions"
```

Comment:
Do you anticipate additional pages? Best to avoid single entry sections, if possible. Can add to the "Contributor guide" below until then

Reply:
Good point, there probably won't be additional pages so the single-entry section may be overkill. Then again, it does make this page stick out which I think is good. What do others think?

Reply:
-1 new section

Reply:
My view is that we should not add this under contributor guide, since the point of adding this is to encourage more feedback from users outside of the contributor community. If we do not want to add a section I would suggest adding this at the bottom of the "further topics" page

Reply:
Moved to further topics!